index.html

```diff
@@ -358,7 +365,7 @@ <h3>Weighing increased fingerprinting surface</h3>
               What browsing contexts, resources and requests need access to a particular feature?  Identifiers can often be scoped to have a different value in different origins. Some configuration may only be necessary in top-level browsing contexts.
             </p>
             <p class="practicedesc">
-              Should access to this functionality be limited to where users have granted a particular permission? While excessive permissions can create confusion and fatigue, limiting highly granular data to situations where a user has already granted permission to access sensitive data widely mitigates the risk of that feature being used primarily for browser fingerprinting in "drive-by" contexts. For example, Media Capture and Streams [[?mediacapture-streams]] limits access to attached microphone and camera device labels to when the user has granted permission to access a camera or microphone (while still allowing access to the number and configuration of attached cameras and microphones in all contexts, a noted increase in drive-by fingerprinting surface).
+              Should access to this functionality be limited to where users have interacted with the document or granted a particular permission? While excessive permissions can create confusion and fatigue, limiting highly granular data to situations where a user has already granted permission to access sensitive data widely mitigates the risk of that feature being used primarily for browser fingerprinting in "drive-by" contexts. For example, Gamepad [[?gamepad]] limits the ability to enumerate attached gamepads until a user has actually interacted with a gamepad in the current document.  Media Capture and Streams [[?mediacapture-streams]] limits access to attached microphone and camera device labels to when the user has granted permission to access a camera or microphone (while still allowing access to the number and configuration of attached cameras and microphones in all contexts, a noted increase in drive-by fingerprinting surface).
```

Comment:
I'm not sure "interacted with the document" is a key criterion for limiting availability of fingerprinting surface, though it might be relevant. Viewing a page, scrolling a page, etc. would be pretty subtle forms of interaction, compared to the examples of granting camera permission or using an attached gamepad device.

Maybe we could just drop this reference from the quick editorial update and then look at different forms of interaction-gating in a separate issue/pr.

Reply:
My thinking here was primarily about ad iframes - I don't think they inherit user interaction from the main document so if a user never interacts with the iframe they wouldn't get the capability.  (At least I hope!)  

But I'm fine  considering this somewhere else - removed it from here.